Common/Product/SharedProject/CommonPackage.cs

```diff
@@ -214,9 +214,11 @@ public abstract class CommonPackage : Package, IOleComponent {
         }
 
         internal static void OpenWebBrowser(string url) {
-            var uri = new Uri(url);
-            Process.Start(new ProcessStartInfo(uri.AbsoluteUri));
-            return;
+            try {
+                var uri = new Uri(url);
+                Process.Start(new ProcessStartInfo(uri.AbsoluteUri));
+            } catch (UriFormatException) {
+            }
```

Comment:
Not ideal. Won't be very common. Some URLs are coming from registry key values, which is where this could happen.